.travis/publish_docs.sh

```diff
@@ -67,13 +80,14 @@ else
 fi
 
 printf "${CYAN}Generating site.$RESET \n"
-mvn clean site
+mvn clean site > /dev/null
```

Comment:
Why are we redirecting the output? What if there is an error and we need to troubleshoot?